src/webui/gitutils.jl

```diff
@@ -226,3 +226,44 @@ mention(u::GitLab.User) = "@$(u.username)"
 # If the registry is from the same provider, mention the user, otherwise use the URL.
 display_user(u::U) where U =
     (parentmodule(typeof(REGISTRY[].repo)) === parentmodule(U) ? mention : web_url)(u)
+
+function tagexists(forge::GitHubAPI, repo::GitHub.Repo, tag::VersionNumber)
+    result = @gf get_tags(forge, repo.owner.login, repo.name)
+
+    if result === nothing
+        @debug("Could not fetch tags")
+        return false
+    end
+
+    for t in result
+        v = split(t.ref, "/")[end]
+        if startswith(v, "v")
+            v = v[2:end]
+        end
+        v == string(tag) && return true
+    end
+    false
```

Comment:
```suggestion
    return false
```

Reply:
I typically use `return` to return and break out of a block. If its the last line I avoid using an extra keyword.

Reply:
Sure, makes sense.